build-managed.cmd

```diff
@@ -0,0 +1,2 @@
+@call %~dp0run.cmd build-managed -nodoReuse -binclashWindows %*
```

Comment:
Maybe I'm misunderstanding how this is used.  I saw the call to `run build-managed` and assumed it was now a top level supported command.  I completely agree it's an implementation detail that shouldn't be exposed.  

But that also makes me not understand this sample.  If it's an implementation detail why is it going through `run.cmd` at all?  Why not skip the middle man and go directly to a `build-managed` command.  What value is `run.cmd` providing here?